recipes/libtar/all/conanfile.py

```diff
@@ -1,14 +1,19 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualRunEnv
+from conan.tools.files import copy, get, replace_in_file, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conan.tools.layout import basic_layout
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.50.0"
 
 
 class LibTarConan(ConanFile):
     name = "libtar"
     description = "libtar is a library for manipulating tar files from within C programs."
-    topics = ("conan", "libtar")
+    topics = ("libtar", "tar")
```

Comment:
Shouldn't the package name be omitted from the topics list?

```suggestion
    topics = ("tar")
```